.vscode/launch.json

```diff
@@ -350,6 +350,22 @@
             // "envFile": "${workspaceRoot}/.env",
             "preLaunchTask": "CDN Serve (Dev)"
         },
+        {
+            /*

```

Comment:
Should it be here or was it just debug ???

Reply:
I added on purpose as I thought this could be useful to more people 😄